lambda_src/lambda_function.py

```diff
@@ -194,19 +194,20 @@ def sync_flow(event: Any, context: Any = None) -> Optional[ResponseType]:
         if k.startswith('sf-custom-')
     }
 
-    res_data = process_batch(
+    result = process_batch(
```

Comment:
the inside of `process_batch` after it calls `process_row` needs to change, as well